src/modules/sharing/models/index.ts

```diff
@@ -3,39 +3,44 @@ import {
   BelongsTo,
   Column,
   DataType,
+  Default,
   ForeignKey,
+  HasMany,
   Model,
   PrimaryKey,
   Table,
 } from 'sequelize-typescript';
 import {
   PermissionAttributes,
   RoleAttributes,
+  SharedWithType,
   SharingAttributes,
   SharingInviteAttributes,
+  SharingActionName,
 } from '../sharing.domain';
 import { UserModel } from '../../user/user.model';
 import { FolderModel } from '../../folder/folder.model';
 import { FileModel } from '../../../modules/file/file.model';
 import { PreCreatedUserModel } from '../../../modules/user/pre-created-users.model';
+import { WorkspaceTeamModel } from '../../workspaces/models/workspace-team.model';
+import { SharingRolesModel } from './sharing-roles.model';
 
 @Table({
   underscored: true,
   timestamps: true,
-  tableName: 'permissions',
```

Comment:
This may be seen as I changed this table, but actually I just switched the table roles and Permissions position because typescript complains about RoleModel not being defined yet.